core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/PurchasesConfiguration.kt

```diff
@@ -13,6 +12,7 @@ public class PurchasesConfiguration private constructor(
     public val appUserId: String?,
     public val purchasesAreCompletedBy: PurchasesAreCompletedBy,
     public val userDefaultsSuiteName: String?,
+    public val storeKitVersion: StoreKitVersion,
```

Comment:
Hmm I would make this nullable, like the `appUserId` or `userDefaultsSuiteName`. This way, we don't need to be duplicating the defaults here.

Reply:
It looks like `IosPurchases.configureWithAPIKey()` requires a non-null `String` to be passed in, so I think it's probably best to keep it as is for now. The default value is `StoreKitVersion.DEFAULT`, which the native iOS SDK is responsible for resolving, so in a roundabout way I guess we're still not duplicating the default version here 😅

Reply:
Oh right, looks like we're using the native configure method for iOS, instead of the PHC one, and that one requires it to be non-null... I guess it's ok then yeah!